gitpod-io/website
# Custom Headers

## Description



Introduced Custom Headers:
- X-Frame-Options
- X-XSS-Protection
- X-Content-Type-Options
- Strict-Transport-Security
- Referrer-Policy
- Content-Security-Policy
- Permissions-Policy


|Before|After|
|:---:|:---:|
|![image](https://user-images.githubusercontent.com/55068936/224251156-b6a5d9aa-2bd4-4323-9557-d97918edc56d.png)|![image](https://user-images.githubusercontent.com/55068936/224251267-4fe8459b-8dba-48ff-945b-01946642398a.png)|

## Related Issue(s)



Related #1690

## How to test


- Open [Preview](https://deploy-preview-3504--gitpod-kumquat.netlify.app/)
- Open Browser Devtools (Go To Network Tab). See following:

![image](https://user-images.githubusercontent.com/55068936/224253059-2e4bd4e4-5aa7-4c04-aba4-b52bbfd71153.png)



Also => 

![image](https://user-images.githubusercontent.com/55068936/224252472-0a1c2119-3006-4f1f-ab10-e077dc38803f.png)


## Release Notes



```release-note
NONE
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3504"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
